### DISCUSSION TITLE
Workflow help

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14461] [General]
I want to try to create  work flow the whenever a product is updated or creating the "product name" should change to its category name....

its works but i am getting a no, rather then the name..  
![image](https://user-images.githubusercontent.com/54880467/112467605-473cbc80-8d8d-11eb-9377-a9011bbff7fa.png)

any idea how to solve this ?

**Comment by rioshaz** [comment]
any help guys ?

**Reply by vovpff** [reply]
Your Workflow returns value what stored in database. This is old and well known bug for me.

You cam implement this task by self written handler

**Reply by rioshaz** [reply]
@vovpff can u please give me steps i can do it,,,, will be great help..

**Reply by vovpff** [reply]
you need php programmer help for this. Or charge for YF team

**Reply by rioshaz** [reply]
@vovpff @mariuszkrzaczkowski  as this is a known bug can we expect a correction..

**Reply by mariuszkrzaczkowski** [reply]
We currently don't have time for that, there are some more pressing matters to take care of right now. The problem will most likely have to wait until someone orders a fix.

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski i understand

**Reply by vovpff** [reply]
Pls write me email I will try to help you

**Reply by rioshaz** [reply]
pls share ur email

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/vovpff

**Reply by rioshaz** [reply]
thanks a lot

**Reply by mariuszkrzaczkowski** [reply]
share the solution to the problem :)

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski sure will do  as i get it ..
